react/src/Datasets/components/AnalysisRunnerDialog.tsx

```diff
@@ -166,7 +159,6 @@ export default function AnalysisRunnerDialog({
                     Cancel
                 </Button>
                 <Button
-                    disabled={!pipeline}
```

Comment:
I should be more clear: I think the button disabling is a good idea, but the asterisk and colouring shouldn't be here unless the dialog explains what they mean.